pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-extractor-utils"
-version = "1.5.4"
+version = "1.5.5"
```

Comment:
There are some other significant changes on master already, so I suggest getting #114 through, and release 1.6.0 🙂

Reply:
ok, I'll wait for it then. So that PR will not do version bump, right?

Reply:
I'll make it 1.6.0 here when #114  goes into master